docs/starknet/ecosystem/sdks/foundry.md

```diff
@@ -66,4 +66,4 @@ $ forge verify-contract <your_contract_address> <path-to-contract>:<contract-nam
 --etherscan-api-key "kakarot_starknet_sepolia" \
 --num-of-optimizations 200 \
 --compiler-version "v0.8.26+commit.8a97fa7a"
-```
\ No newline at end of file
+```
```

Comment:
what is the change made here?

Reply:
nothing has changed here, the only thing is a typo